### ISSUE TITLE
About dataset format

### ISSUE THREAD

**Issue by songlin** [ac-rad/MVTrans#10]
Dear author,

I want to test our method on your datasets. However, we are confused about the dataset, particularly the depth map.

We downloaded the pickle files and tried to extract the depth map as follows:

```
uid = 11840 # e.g, 
h = LocalReadHandle(download_path, int(uid))
datapoint = h.read()
depth = datapoint.depth

left_image = datapoint.stereo.left_color 
right_image = datapoint.stereo.right_color

# convert depth to disparity
f = datapoint.camera_params['camera_intrinsic'][0][0] # 613.9624633789062
b = datapoint.camera_params['baseline'] # 0.06499999761581421
disp = f * b / depth

# pseudo code: warp right image according to the disparity
warpped_left = warp(right_image, disp) 
loss = F.l1_loss(warpped_left, left)
```

We found the loss is pretty high, meaning the disparity could be very wrong.
So, how could we get the depth, and camera intrinsics right?

Thank you in advance for the help!
